flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -35,23 +35,48 @@ function setup_kubernetes_for_linux {
     else
         local arch='amd64'
     fi
+
     # Download kubectl, which is a requirement for using minikube.
     if ! [ -x "$(command -v kubectl)" ]; then
         echo "Installing kubectl ..."
         local version=$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)
         curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$version/bin/linux/$arch/kubectl && \
             chmod +x kubectl && sudo mv kubectl /usr/local/bin/
     fi
-    # Download minikube when it is not installed beforehand.
-    if [ -x "$(command -v minikube)" ] && [[ "$(minikube version | grep -c $MINIKUBE_VERSION)" == "0" ]]; then
-      echo "Removing any already installed minikube binaries ..."
-      sudo rm "$(which minikube)"
-    fi
 
-    if ! [ -x "$(command -v minikube)" ]; then
-      echo "Installing minikube $MINIKUBE_VERSION ..."
-      curl -Lo minikube https://storage.googleapis.com/minikube/releases/$MINIKUBE_VERSION/minikube-linux-$arch && \
-          chmod +x minikube && sudo mv minikube /usr/bin/minikube
+    if [[ ${NO_SPECIFY_MINIKUBE_VERSION} == false ]]; then
```

Comment:
can't we just use "latest" as a valid value if we want to support the latest version? :thinking: In that case we would remove the currently installed version (if it exists) and reinstall the desired (latest) version of Minikube. That makes the code way easier (no version parsing which might introduce new issues down the road). WDYT?